tests/integration/test_github/conftest.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import pytest
+
+from wazuh_testing.constants.paths.logs import OSSEC_LOG_PATH
```

Comment:
Pull latest changes from framework and update `OSSEC_LOG_PATH` to `WAZUH_LOG_PATH`

Reply:
Change applied with [this commit](https://github.com/wazuh/wazuh/pull/17621/commits/42030e38bc82fa2fb6e842adf8abee6b919cfeef)